### ISSUE TITLE
Settings page should scroll

### ISSUE THREAD

**Issue by gnprice** [zulip/zulip-flutter#1904]
The settings page should be scrollable if the content doesn't all fit on the screen.

Currently there are only a few settings, so for most users this doesn't happen. But it can happen if the user has a very small screen (https://github.com/zulip/zulip-flutter/issues/1898#issuecomment-3386032763) or have their display zoomed in for accessibility. In the future it'll probably happen for more users, as we add more settings.

**Comment by sandy4242** [comment]
@gnprice hey hi, could i be assigned to this issue? 
make the setting screen scrollable would be a great start for me to be a good contributor for this community...
Thank you...

**Comment by chrisbobbe** [comment]
This issue is already being worked on; see #1911.

**Comment by sandy4242** [comment]
@gnprice ohh yes thank you, didn't see the commit reference...
